config/publishers/bitbucket.md

```diff
@@ -16,15 +16,15 @@ Full configuration options are documented in [`PublisherBitbucketConfig`](https:
 
 ```javascript
 {
-  "name": "@electron-forge/publisher-bitbucket",
-  "config": {
-    "repository": {
-      "owner": "myusername",
-      "name": "myreponame"
+  name: "@electron-forge/publisher-bitbucket",
```

Comment:
Three things here:
* We usually use single-quotes for JS
* Can we add the `forge.config.js` code snippet title? There's some GitBook-specific syntax that we can use here. See: https://github.com/electron-forge/electron-forge-docs/blob/v6/guides/code-signing/code-signing-macOS.md
* We might want to expand on this a bit by adding an abridged version of the full JS object in `forge.config.js`. Something like the below

```js
module.exports = {
  // ...
  publishers: [
     {
       name: 'asldjkas;d';ad",
       config: { }
     }
  ]

}
```

Reply:
good catch, in my fever to remove the quotes from the left side i forgot to look at the right